lumigator/frontend/src/components/molecules/LDatasetEmpty.vue

```diff
@@ -1,10 +1,15 @@
 <template>
   <div class="l-dataset-empty">
-    <h2 class="l-dataset-empty__instructions">Submit a dataset to start using Lumigator.</h2>
+    <h2 class="l-dataset-empty__instructions">Use a dataset as the basis for your evaluation. It includes data for the model you'd like to
+      evaluate and possibly a ground truth "answer".</h2>
     <p class="l-dataset-empty__instructions-text">
       <span>
-        Use a dataset as the basis for your evaluation. It includes data for the model you'd like to
-        evaluate and possibly a ground truth "answer".
+        If you don't have your dataset yet, you can use our
+        <a
+        href="https://github.com/mozilla-ai/lumigator/tree/main/lumigator/sample_data"
+        target="_blank"
+        style="background-color: transparent"
+        > <span class="underline">sample datasets</span> <span class="pi pi-arrow-up-right" /></a>
```

Comment:
nit: `<span>` tags are not self-closing in html unlike `<a>` tags

```suggestion
        > <span class="underline">sample datasets</span> <span class="pi pi-arrow-up-right"></span></a>
```